kornia/augmentation/_2d/intensity/random_rain.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict, Optional, Tuple
+
+import torch
+
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.augmentation.random_generator._2d import RainGenerator
+from kornia.core import Tensor
+from kornia.core.check import KORNIA_CHECK
+
+
+class RandomRain(IntensityAugmentationBase2D):
+    r"""Add Random Rain to the image.
+
+    Args:
+        p: probability of applying the transformation.
+        number_of_drops: number of drops per image
+        drop_height: height of the drop in image(same for each drops in one image)
+        drop_width: width of the drop in image(same for each drops in one image)
+    Shape:
+        - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`, Optional: :math:`(B, 3, 3)`
```

Comment:
Deprecated description.
```suggestion
        - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`
```